examples/imageviewer/webApp/src/jsMain/kotlin/Main.js.kt

```diff
@@ -0,0 +1,12 @@
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.window.CanvasBasedWindow
+import org.jetbrains.skiko.wasm.onWasmReady
+
+@OptIn(ExperimentalComposeUiApi::class)
+fun main() {
+    onWasmReady {
+        CanvasBasedWindow("ImageViewer") {
```

Comment:
a TODO for us: Prepare a follow up PR with an update to compose 1.6.10-beta03 or 1.6.10-rc01, where we'll use a new API refined - `ComposeViewport`.

For now, this update is not required, because we try to keep only stable versions in the examples in the main branhc